solr/api/build.gradle

```diff
@@ -17,6 +17,7 @@
 
 plugins {
   id 'io.swagger.core.v3.swagger-gradle-plugin' version '2.2.2'
+  id "org.openapi.generator" version "6.0.1"
```

Comment:
BTW, I looked and there is a 7.0.1 versin https://plugins.gradle.org/plugin/org.openapi.generator, maybe we have to update gradle?